periodic/periodic.go

```diff
@@ -546,6 +556,16 @@ func formatDate(d *time.Time) string {
 		d.Hour(), d.Minute(), d.Second())
 }
 
+// Return a jitter time that is (+/-)10% of the duration t
+func addJitter(t time.Duration) time.Duration {
+	if t <= 0 {
```

Comment:
make that if t < 10 as for small values it will anyway turn into i == 0?

Reply:
there are a few cases I observed: t could be a negative time.Duration value if a thread is falling behind. In that case, we don't want to add more jitter to worsen the situation, so the function would return 0 jitter. t could also be super small, i.e., t < 10 would mean it's less than 10ns. In that case, returning 0 jitter shouldn't matter one or another.

Reply:
just meant to avoid calculating things that will end up being a 0 jitter (integer division of 9/10 -> 0, then Rand(2*0) is 0)